### DISCUSSION TITLE
Import data from Randomly RemindMe

### DISCUSSION THREAD

**Discussion by krzysztofjeziorny** [iSoron/uhabits#217] [Feature Requests]
I can't find the info for right format of Habits' CSV file, and as I don't have my own yet I thought I maybe post a feature request that would be also of interest to others.

For some time I used a quite nice app Randomly Remind Me, now I'd like to try out Habits with my stored data. RRM exports the log as CSV, an example is attached as zip. Thank you!

[RRM_csv_export_log.csv.zip](https://github.com/iSoron/uhabits/files/629738/RRM_csv_export_log.csv.zip)

**Comment by iSoron** [comment]
Thank you for the suggestion, @TravellerSam.
